lib/Slic3r/GUI/ReloadDialog.pm

```diff
@@ -12,16 +12,16 @@ use base 'Wx::Dialog';
 sub new {
     my $class = shift;
     my ($parent,$default_selection) = @_;
-    my $self = $class->SUPER::new($parent, -1, "Additional parts and modifiers detected", wxDefaultPosition, [350,100], wxDEFAULT_DIALOG_STYLE);
+    my $self = $class->SUPER::new($parent, -1, "Additional parts/modifiers detected", wxDefaultPosition, [350,100], wxDEFAULT_DIALOG_STYLE);
     
     # label
-    my $text = Wx::StaticText->new($self, -1, "Additional parts and modifiers are loaded in the current model. \n\nHow do you want to proceed?", wxDefaultPosition, wxDefaultSize);
+    my $text = Wx::StaticText->new($self, -1, "Additional parts or modifiers were added in Slic3r to the model you're reloading. \n\nHow do you want to proceed?", wxDefaultPosition, wxDefaultSize);
 
     # selector
     $self->{choice} = my $choice = Wx::Choice->new($self, -1, wxDefaultPosition, wxDefaultSize, []);
-    $choice->Append("Reload all linked files");
-    $choice->Append("Reload main file, copy added parts & modifiers");
-    $choice->Append("Reload main file, discard added parts & modifiers");
+    $choice->Append("Reload parts/modifiers");
+    $choice->Append("Keep parts/modifiers (don't reload)");
+    $choice->Append("Discard parts/modifiers");
     $choice->SetSelection($default_selection);
```

Comment:
With the addition of 'prompt', `$default_selection` may not be in the choice dropdown, which is causing a warning.